src/pages/sessions/styles.ts

```diff
@@ -15,7 +15,11 @@
     You should have received a copy of the GNU General Public License
     along with this program.  If not, see <https://www.gnu.org/licenses/>.
 */
-import { makeStyles } from "@material-ui/core";
+
+//!!!!!!!!! TO BE REPLACED WITH styled() WHEN DOING "feat: style session table in /sessions #94" GITHUB ERROR
+// (look for other styles.ts files as an example)
```

Comment:
I'm not entirely sure what you mean here with the GithHub error.

Reply:
Because this page( /sessions ) right now does not use its own styling (uses dashboard/session styling) and github issue #94 suggest to change it so that it will use its own styling. Thus, currently this file is not in use and I didnt change anything there. If needed I can change those stylings to v5 styled.

Reply:
Oh I see. Thanks for that.